mlflow/tracking/llm_utils.py

```diff
@@ -0,0 +1,93 @@
+import csv
+import logging
+import mlflow
+import os
+import shutil
+import tempfile
+
+from typing import Dict, List, Union
+
+_logger = logging.getLogger(__name__)
+
+
+def log_predictions(
+    inputs: List[Union[str, Dict[str, str]]],
+    outputs: List[str],
+    prompts: List[Union[str, Dict[str, str]]],
```

Comment:
Let's say I have a list of questions and answers. To log them, do I need to make a list of empty strings (`[""] * N`) for `propmts`?

Reply:
Yes I think right now, we don't think the prompt is optional. We can make it optional later on if we see that it is necessary. What do you think?

Reply:
Sounds good!